lib/src/gestures/interactive_flag.dart

```diff
@@ -39,4 +42,25 @@ class InteractiveFlag {
   static bool hasFlag(int leftFlags, int rightFlags) {
     return leftFlags & rightFlags != 0;
   }
```

Comment:
If we now have individual checkers, should `hasFlag` even be public anymore?

Reply:
`hasFlag` lets you check multiple flags so I think we should keep it. Maybe we should rename it to hasFlags to make that clearer (with a redirecting method and deprecation).

Reply:
I wouldn't bother then :D